tests/cli/experiment/delete_test.py

```diff
@@ -0,0 +1,128 @@
+from click.testing import CliRunner
+import unittest
+from mock import patch, call
+
+import floyd
+from floyd.cli.experiment import delete
+from .mocks import mock_exp, mock_running_exp, mock_queued_exp, mock_task_inst
```

Comment:
One minor detail: We don't use relative imports in the code anywhere else. So for consistency, this should be switched to full module name.